src/rover_control/rover_control.hpp

```diff
@@ -0,0 +1,29 @@
+#include <network.hpp>
+#include "drive_controller.hpp"
```

Comment:
The drive controller include shouldn't be necessary. For the mode setter, it can use `drive_msg::DriveMode`. This is already done in the cpp file but the set_drive_mode function specified below uses the wrong one

Also you will need `rover_system_messages.hpp` here